app/assets/stylesheets/components/_theme.scss

```diff
@@ -0,0 +1,71 @@
+$Theme-spacing-base: 18px;
+$Theme-spacing-small: 9px;
+
+$Theme-fontSize-base: 18px;
+$Theme-fontSize-large: 22px;
+
+$Theme-height-header: 100px;
```

Comment:
Let's try to share as little things as we can between components.
In this case, this means: Move the header height into the Header component, only he needs to know it.